data/presets/natural/tree.json

```diff
@@ -5,11 +5,17 @@
         "leaf_cycle_singular",
         "denotation",
         "height",
-        "diameter"
+        "circumference"
```

Comment:
Trees are conventionally measured by [diameter at breast height](https://en.wikipedia.org/wiki/Diameter_at_breast_height). However, [`circumference`](https://wiki.openstreetmap.org/wiki/Key:circumference) is the more prominently documented way to encode that information. Perhaps the idea is that it’s easier to get the circumference on a field survey using a measuring tape without having to employ trigonometry. Should we clarify the name of this field and add “diameter at breast height” and “dbh” as search terms for it? If so, we’ll probably need to split off a separate field, to avoid affecting other presets that coincidentally contain a Circumference field.

https://github.com/openstreetmap/id-tagging-schema/blob/be3954f8cc16b3150f9eed668a04b1aa9038a76f/data/fields/circumference.json#L4

Reply:
currently only the tree preset uses this field, so we're fine. :wink: 

updated in 2c6c044